fastprogress/fastprogress.py

```diff
@@ -254,13 +255,14 @@ def update_graph(self, graphs, x_bounds=None, y_bounds=None, figsize=(6,4)):
 
 class ConsoleProgressBar(ProgressBar):
     fill:str='█'
+    end:str='\r'
 
     def __init__(self, gen, total=None, display=True, leave=True, parent=None, auto_update=True, txt_len=60):
         self.cols,_ = shutil.get_terminal_size((100, 40))
         if self.cols > MAX_COLS: self.cols=MAX_COLS
         self.length = self.cols-txt_len
         self.max_len,self.prefix = 0,''
-        try: print(self.fill, end='\r')
+        try: print(self.fill, end='\r', flush=FLUSH)
```

Comment:
Shoudn't it be end=self.end here?

Reply:
I've done some additional testing, and IMO the "nicest" behavior is when we clear the last progress bar line with `\r` after whole iteration process is finished. This prevents overlapping log lines, and also clears finished (100%) progress bars in console.

Alternatively, we could end the line with `\n`.